packages/create-rsbuild/src/index.ts

```diff
@@ -161,7 +161,8 @@ async function getTools({ tools, dir, template }: Argv) {
 
   return checkCancel<string[]>(
     await multiselect({
-      message: 'Select additional tools (press enter to continue)',
+      message:
+        'Select additional tools (Use <space> to select, <enter> to continue)',
```

Comment:
Thanks! Can you update `/guide/start/quick-start.mdx` too?

https://github.com/web-infra-dev/rsbuild/blob/f6d991fa2fa7af0d8ec9645538a6a353969d8c68/website/docs/en/guide/start/quick-start.mdx?plain=1#L67